creator/ingest_runs/queries/validation_run.py

```diff
@@ -24,14 +27,40 @@ class Query(object):
         filterset_class=ValidationRunFilter,
         description="Get all validation_runs",
     )
+    last_modified_active_validation_run = Field(
+        ValidationRunNode,
+        description=(
+            "Get the last modified active validation_run for a data_review"
+        ),
+        dr_id=String(required=True),
+    )
 
     def resolve_all_validation_runs(self, info, **kwargs):
         """
         Return all validation_runs
         """
         user = info.context.user
-
         if not user.has_perm("ingest_runs.list_all_validationrun"):
             raise GraphQLError("Not allowed")
 
         return ValidationRun.objects.all()
+
+    def resolve_last_modified_active_validation_run(
+        self, info, dr_id, **kwargs
+    ):
+        """
+        Return the last modified validation_run for the data_review with
+        kf_id=dr_id which is currently active (i.e. not completed, canceled, or
+        failed).
+        """
+        user = info.context.user
+        if not user.has_perm("ingest_runs.list_all_validationrun"):
+            raise GraphQLError("Not allowed")
+
+        bad_states = {State.CANCELED, State.COMPLETED, State.FAILED}
+        return (
+            ValidationRun.objects.filter(data_review__kf_id=dr_id)
+            .exclude(state__in=bad_states)
+            .order_by("-modified_at")
```

Comment:
I'm actually not totally sure. I'm working off the issue which says:
`The frontend needs to be able to easily fetch a data review's last active / last modified validation run when it polls the backend for updates on the active validation run.`

@znatty22 ?

Reply:
This feature is desired because right now the frontend queries for all of the validation runs for a review and then sorts them on the frontend to determine which run is active (not in a terminal state) and the most recent. It does this when it polls the backend for the validation run status so it knows what kind of loader (e.g. Validating, Canceling, etc) to display to the user. 

I believe we still want this feature, but let's ask @XuTheBunny?